src/urbit/app/chess.hoon

```diff
@@ -854,7 +857,7 @@
       =/  fen  (position-to-fen position.game-state)
       =/  cards  ^-  (list card)
         :~  :*  %give  %fact  ~[/game/(scot %da u.game-id)/updates]
-                %chess-update  !>([%position u.game-id fen special-draw-available.game-state])
+                %chess-update  !>([%position u.game-id fen special-draw-available.game-state ~])
```

Comment:
Is this card necessary? If we send the entire array of moves with the game info on start-up (each of which should come with its resulting FEN position), do we even need to send the latest position?

I guess we don't need it now, but it would be helpful for spectating mode, in the future (#67). Just thinking out loud here, not an issues to address.

Reply:
See new comment at this line in latest review